src/Components/OATHeader/internal/FileSubMenu.tsx

```diff
@@ -205,7 +206,11 @@ export const FileSubMenu = ({
                 setIsFileStored(true);
             }
         }
-    }, []);
+    };
+
+    useEffect(() => {
+        onProjectChange();
+    }, [projectName, isActive]);
```

Comment:
Why is `isActive` a dependency here? It doesn't appear to be used in the `onProjectChange` function?